src/rgw/rgw_trim_datalog.cc

```diff
@@ -128,9 +128,13 @@ int DataLogTrimCR::operate()
         ldout(cct, 10) << "trimming log shard " << i
             << " at marker=" << m
             << " last_trim=" << last_trim[i] << dendl;
-        spawn(new TrimCR(store, store->svc()->datalog_rados->get_oid(i),
-                         m, &last_trim[i]),
-              true);
+#warning BOOM!
```

Comment:
boom?

Reply:
This gets fixed in a later commit.

Reply:
I love comments like this, definitely should keep! :smile: